editions/tw5.com/tiddlers/howtos/Formatting List Results as Tables with CSS - Specified Columns Methods.tid

```diff
@@ -17,13 +17,9 @@ For other table-making techniques see also:
 ```
 /* FOUR COLUMN MODE */
 .fourcolumns {
-   display:block;
-   column-count:4;
-   column-gap:1em;
-   -moz-column-count:4;
-   -moz-column-gap:1em;
-   -webkit-column-count: 4;
-   -webkit-column-gap:1em;
+   display: block;
```

Comment:
It's not clear why these changes belong in this PR.

Reply:
The style is hardcoded with a `<style>` attribute in this tiddler and it's not globally available. 

I do need the `.fourcolumns` style definition in a different tiddler whis is part of this PR. That's why this definition was moved to the `editions/tw5.com/tiddlers/system/tw5.com-styles.tid` stylesheet also part of this PR, to make it globally available.

Reply:
- Neither https://caniuse.com/?search=column-gap
- nor https://caniuse.com/?search=column-count  

need a prefix anymore. So I think it can be safely removed